.github/workflows/cd.yml

```diff
@@ -0,0 +1,20 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    types:
+      - completed
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+permissions:
+  contents: read
+
+env:
```

Comment:
This is not needed actually, it is always true in github actions. check for reference https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Thanks, I wasn't 100% sure so I wanted to be on the safe side but looks like it can be removed indeed. Will update.